crates/tako/src/internal/server/rpc.rs

```diff
@@ -95,10 +93,6 @@ pub(crate) async fn worker_authentication(
         &mut reader,
     )
     .await?;
```

Comment:
The `assert` wasn't useful?

Reply:
That if you have `Some(key)`, `do_authentication` returned `Some(sealer)`. But it's probably not needed.